package.json

```diff
@@ -137,6 +137,11 @@
 								"type": "boolean",
 								"description": "Enable external display for debug",
 								"default": false
+							},
```

Comment:
I'd suggest to change `gdbtty` to use:

```ts
								"type": [
									"boolean",
									"string"
								],
```

then you have `false` (= old behaviour - part of the debug console), `true` (either `"external"` or, which I'd prefer mean `"vscode",`) or  one of the texts `"external"`/ `"vscode"

I think we don't need an explicit "xterm", do we?
If using win32 either forever or currently doesn't support `"vscode"`, then we can just raise an error in this cas, or a warning and fallback to `"external"`.

Reply:
OK. It was as you suggested: true, false, "vscode", "external".